app/domain/logs.rb

```diff
@@ -658,6 +658,16 @@ module AuthnJwt
         msg: "Successfully fetched mapping claims '{0-mapping-claims}'",
         code: "CONJ00130I"
       )
+
+      CreateContraintsFromPolicy = ::Util::TrackableLogMessageClass.new(
+        msg: "Creating constraints from policy...",
```

Comment:
challenge  the "from policy" term

Reply:
@hilagross @shulifink 
We did agree we call it from policy in all places not?